src/components/customer/selectService/selectService.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from "react";
+import "@css/customer/selectService/selectServiceComponent.css";
+
+// Defining the properties received by the component
+interface ServiceProps {
+  onSelectService: (serviceId: string, servicePrice: number) => void; // Receives ID and Price
+}
+
+// Structure of the service data
+interface Service {
+  id: number;
+  name: string;
+  description: string | null;
+  duration: number;
+  price: number;
+  storeId: number;
+}
+
+const Service: React.FC<ServiceProps> = ({ onSelectService }) => {
+  const [data, setData] = useState<Service[] | null>(null);
+
+  // Using useEffect to load data from the API
+  useEffect(() => {
```

Comment:
This API call should not be here, it should be in another file, check clean architecture for react and see where this code should be located

Reply:
It's already fixed